files/image_config/platform/rc.local

```diff
@@ -29,7 +29,11 @@ if [ -f /host/image-$sonic_version/platform/firsttime ]; then
         exit 0
     fi
 
-    cp /usr/share/sonic/device/$platform/minigraph.xml /etc/sonic/
+    if [ -f /host/minigraph.xml ]; then
+        mv /host/minigraph.xml /etc/sonic/
```

Comment:
If it will be better to add some comments here. In the future we could remove this logic when old image with /host/minigraph.xml is deprecated.